### ISSUE TITLE
[Enhancement]: Import `aws_security_group_rule` via Security Group Rule ID

### ISSUE THREAD

**Issue by philnielsen** [hashicorp/terraform-provider-aws#29404]
### Description

Now that security group rules have [native AWS ID's ](https://aws.amazon.com/blogs/aws/easily-manage-security-group-rules-with-the-new-security-group-rule-id/), I'd like to import via that instead of via the convoluted string that is currently used. I'm not sure how this would work with the provider internals as there seems to already be a rule ID built into the provider. I tried to find other issues referencing this documentation but couldn't find any.

### Affected Resource(s) and/or Data Source(s)

aws_security_group_rule

### Potential Terraform Configuration

```terraform
terraform import aws_security_group_rule.rule_name sgr-abcdefghi01234561
```

### References

https://aws.amazon.com/blogs/aws/easily-manage-security-group-rules-with-the-new-security-group-rule-id

### Would you like to implement a fix?

No

**Comment by ewbankkit** [comment]
Relates https://github.com/hashicorp/terraform-provider-aws/issues/20104.

**Comment by ewbankkit** [comment]
This will be addressed via the new Security Group Rule resources implemented in #29484.
